node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,9 @@
+import fs from 'node:fs/promises';
+
+async function readAny() {
+  const filePath = `./${process.argv[2]}`;
```

Comment:
The `./` isn't necessary and makes this more restrictive than it should be. It's best to let the user type the path on the command line.

As it turns out, readFile starts looking in the current directory, so, for example, just `dijkstra.txt` will work, with the leading `./`.